<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>calib_dialog</class>
    <widget class="QDialog" name="calib_dialog">
        <property name="windowModality">
            <enum>Qt::ApplicationModal</enum>
        </property>
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>600</width>
                <height>371</height>
            </rect>
        </property>
        <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
            </sizepolicy>
        </property>
        <property name="minimumSize">
            <size>
                <width>600</width>
                <height>350</height>
            </size>
        </property>
        <property name="maximumSize">
            <size>
                <width>1200</width>
                <height>700</height>
            </size>
        </property>
        <property name="windowTitle">
            <string>Calibration - MIDIBrain</string>
        </property>
        <property name="modal">
            <bool>true</bool>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
            <item>
                <layout class="QVBoxLayout" name="verticalLayout_2">
                    <item>
                        <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,1,0">
                            <item>
                                <spacer name="horizontalSpacer">
                                    <property name="orientation">
                                        <enum>Qt::Horizontal</enum>
                                    </property>
                                    <property name="sizeHint" stdset="0">
                                        <size>
                                            <width>40</width>
                                            <height>20</height>
                                        </size>
                                    </property>
                                </spacer>
                            </item>
                            <item>
                                <widget class="QLabel" name="label">
                                    <property name="sizePolicy">
                                        <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
                                            <horstretch>0</horstretch>
                                            <verstretch>0</verstretch>
                                        </sizepolicy>
                                    </property>
                                    <property name="lineWidth">
                                        <number>1</number>
                                    </property>
                                    <property name="text">
                                        <string>
                                        The calibration process calculates the lower and upper limits of the Alpha and Beta waves.
                                        The calibration for each band (Alpha or Beta) starts as soon as you press the respective button and takes one minute.
                                        Upon completion of the calibration you will hear a sound.
                                        </string>
                                    </property>
                                    <property name="textFormat">
                                        <enum>Qt::RichText</enum>
                                    </property>
                                    <property name="scaledContents">
                                        <bool>false</bool>
                                    </property>
                                    <property name="alignment">
                                        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                                    </property>
                                    <property name="wordWrap">
                                        <bool>true</bool>
                                    </property>
                                    <property name="indent">
                                        <number>-1</number>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <spacer name="horizontalSpacer_2">
                                    <property name="orientation">
                                        <enum>Qt::Horizontal</enum>
                                    </property>
                                    <property name="sizeHint" stdset="0">
                                        <size>
                                            <width>40</width>
                                            <height>20</height>
                                        </size>
                                    </property>
                                </spacer>
                            </item>
                        </layout>
                    </item>
                </layout>
            </item>
            <item>
                <widget class="Line" name="line">
                    <property name="orientation">
                        <enum>Qt::Horizontal</enum>
                    </property>
                </widget>
            </item>
            <item>
                <layout class="QHBoxLayout" name="horizontalLayout_4">
                    <property name="spacing">
                        <number>40</number>
                    </property>
                    <property name="leftMargin">
                        <number>20</number>
                    </property>
                    <property name="rightMargin">
                        <number>20</number>
                    </property>
                    <item>
                        <layout class="QVBoxLayout" name="verticalLayout_3">
                            <property name="spacing">
                                <number>15</number>
                            </property>
                            <item>
                                <widget class="QLabel" name="alpha_calib_label">
                                    <property name="sizePolicy">
                                        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                                            <horstretch>0</horstretch>
                                            <verstretch>0</verstretch>
                                        </sizepolicy>
                                    </property>
                                    <property name="text">
                                        <string>
                                        Close your eyes and relax, without thinking about anything particular
                                        </string>
                                    </property>
                                    <property name="textFormat">
                                        <enum>Qt::RichText</enum>
                                    </property>
                                    <property name="alignment">
                                        <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
                                    </property>
                                    <property name="wordWrap">
                                        <bool>true</bool>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QPushButton" name="a_calib_button">
                                    <property name="sizePolicy">
                                        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                                            <horstretch>0</horstretch>
                                            <verstretch>0</verstretch>
                                        </sizepolicy>
                                    </property>
                                    <property name="text">
                                        <string>Calibrate Alpha waves</string>
                                    </property>
                                    <property name="autoDefault">
                                        <bool>false</bool>
                                    </property>
                                    <property name="flat">
                                        <bool>false</bool>
                                    </property>
                                </widget>
                            </item>
                        </layout>
                    </item>
                    <item>
                        <layout class="QVBoxLayout" name="verticalLayout_4">
                            <property name="spacing">
                                <number>15</number>
                            </property>
                            <item>
                                <widget class="QLabel" name="beta_calib_label">
                                    <property name="sizePolicy">
                                        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                                            <horstretch>0</horstretch>
                                            <verstretch>0</verstretch>
                                        </sizepolicy>
                                    </property>
                                    <property name="text">
                                        <string>
                                        Keep your eyes open this time and try to really focus on something.
                                        E.g. count the alphabet backwards.
                                        </string>
                                    </property>
                                    <property name="textFormat">
                                        <enum>Qt::RichText</enum>
                                    </property>
                                    <property name="alignment">
                                        <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
                                    </property>
                                    <property name="wordWrap">
                                        <bool>true</bool>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QPushButton" name="b_calib_button">
                                    <property name="sizePolicy">
                                        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                                            <horstretch>0</horstretch>
                                            <verstretch>0</verstretch>
                                        </sizepolicy>
                                    </property>
                                    <property name="text">
                                        <string>Calibrate Beta waves</string>
                                    </property>
                                    <property name="autoDefault">
                                        <bool>false</bool>
                                    </property>
                                </widget>
                            </item>
                        </layout>
                    </item>
                </layout>
            </item>
            <item>
                <widget class="Line" name="line_2">
                    <property name="orientation">
                        <enum>Qt::Horizontal</enum>
                    </property>
                </widget>
            </item>
            <item>
                <widget class="QProgressBar" name="cal_progbar">
                    <property name="enabled">
                        <bool>false</bool>
                    </property>
                    <property name="value">
                        <number>0</number>
                    </property>
                    <property name="textVisible">
                        <bool>true</bool>
                    </property>
                    <property name="invertedAppearance">
                        <bool>false</bool>
                    </property>
                    <property name="format">
                        <string>%p%</string>
                    </property>
                </widget>
            </item>
        </layout>
    </widget>
    <resources/>
    <connections/>
</ui>
